cfpb/cfgov-refresh
# Converting to info-unit throughout and removing extra styles/markup

Converting to info-unit throughout and removing extra styles/markup  …

## Additions

- Add Info Unit Macro

## Removals

- Remove Link Blob, 25/75, and 50/50 styles
- Remove need for negative margin tweaks after groups
- Remove need for positive margin tweaks aroung group headings

## Changes
- Convert Link Blob Group, 25/75 Group, and 50/50 Group to single Info Unit Group
- Convert Link Blob Macro to Info Unit Macro
- Convert  25/75 Macro to Info Unit Macro
- Convert  50/50 Macro to Info Unit Macro
- Update Home Page to Info Unit Macro

## Testing

- `gulp build`
- create a new Wagtail page with multiple Link Blobs, 25/75s, and 50/50s
- check out home page

## Review

- @KimberlyMunoz 
- @anselmbradford 
- @sebworks 

## Screenshots

<img width="710" alt="screen shot 2016-02-19 at 5 04 10 pm" src="https://cloud.githubusercontent.com/assets/1280430/13190366/d1e19fbe-d72a-11e5-9a75-fd7cab2a8b79.png">

<img width="748" alt="screen shot 2016-02-19 at 5 04 22 pm" src="https://cloud.githubusercontent.com/assets/1280430/13190375/d83a3682-d72a-11e5-8634-3b31c0a4636c.png">

<img width="749" alt="screen shot 2016-02-19 at 5 04 34 pm" src="https://cloud.githubusercontent.com/assets/1280430/13190378/dfc09842-d72a-11e5-8cde-690e175df66c.png">

_I didn't have a square image on my machine, Ned has to suffice and is a bit stretched, but only used for a layout example_

## Notes

- I've learned the hard way `.content-l` and `.content-l_col` shouldn't be combined w/ other classes, this leads to anger, anger leads to hate, hate leads to suffering.
- The only children of `.content-l` should be `.content-l_col`. Anything else, well, see above.
- Only square images should be decorative, anything else should be a real image.
- I don't know why, but we've forced `col-1` to remove any margin in cf-enhancements. I can't find anywhere else that's set, so I'd like to remove it to remove the `!important` in the `o-info-unit-group`

## Todos

- Convert CMS to `is_square` checkbox instead of `is_16x9`

## Checklist
